packages/newsroom-signup/src/NewsroomProfile/LearnMoreButton.tsx

```diff
@@ -78,45 +78,78 @@ export class LearnMoreButton extends React.Component<LearnMoreButtonProps, Learn
           <BorderlessButton onClick={this.close}>Back</BorderlessButton>
           <ContentWrapper>
             <Header>About applying to join Civil</Header>
-            <SectionHeader>What is the process and what to expect</SectionHeader>
             <Paragraph>
-              In this section we'll take you thought the steps of completing your application to join the Civil network.
-              You will be led through the steps of completing your application to the Civil Registry, which unlocks
-              several key benefits. It allows you to be part of a community that has vetted your newsroom and deemed it
-              as having a credible journalistic mission. It also allows your content to be featured alongside content
-              from other approved Civil network newsrooms.
+              In this section we'll take you through the steps to join the Civil network. But first, let’s start with a
+              few key terms. In this section we'll take you through the steps to join the Civil network. But first,
+              let’s start with a few key terms.
```

Comment:
looks like this sentence repeats

Reply:
yeah, you are right. i removed it from my end